buildbot.mariadb.org/scripts/rpm-install.sh

```diff
@@ -1,25 +1,39 @@
 #!/usr/bin/env bash
 # shellcheck disable=SC2154
 
-set -ex
-# //TEMP why vlad?
-branch=${1:-bb-10.8-release}
+set -e
+
+# Buildbot installation test script
+# this script can be called manually by providing the build URL as argument:
+# ./rpm-install.sh "https://buildbot.mariadb.org/#/builders/368/builds/695"
+
+# load common functions
+# shellcheck disable=SC1091
+. ./bash_lib.sh
+
+# function to be able to run the script manually (see bash_lib.sh)
+manual_run_switch "$1"
+
+set -x
+
+# print disk usage
 df -kT
 
-# //TEMP this should be done in the VM preparation IMO
-case "$branch" in
-  *mdev10416*)
-    sudo cat /etc/sysconfig/selinux | grep SELINUX || true
-    sudo sh -c \"PATH=$PATH:/usr/sbin getenforce || true\"
-    sudo sh -c \"PATH=$PATH:/usr/sbin setenforce Enforcing || true\"
-    sudo sh -c \"PATH=$PATH:/usr/sbin getenforce || true\"
-    ;;
-esac
+# # //TEMP this should be done in the VM preparation
+# case "$master_branch" in
```

Comment:
here I guess it should still be $branch since $master_branch will always be only 10.x, But since it's already commented it doesn't have much impact

Reply:
[bb release branches trigger install too](https://buildbot.mariadb.org/#/builders/198/builds/1823).